api/store/mongo/migrations/migration_23.go

```diff
@@ -0,0 +1,29 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var migration_23 = migrate.Migration{
+	Version:     23,
+	Description: "convert names and emails to lowercase",
+	Up: func(db *mongo.Database) error {
+		logrus.Info("Applying migration 23 - Up")
+		_, err := db.Collection("namespaces").UpdateMany(context.TODO(), bson.D{}, []bson.M{
+			{
+				"$set": bson.M{"name": bson.M{"$toLower": "$name"}},
+			},
```

Comment:
Isn't this lacking the email rework as well?

Reply:
done